@xen-orchestra/backups/_runners/_vmRunners/IncrementalRemote.mjs

```diff
@@ -114,6 +114,10 @@ class IncrementalRemoteVmBackupRunner extends AbstractRemote {
       await this._callWriters(writer => writer.cleanup(), 'writer.cleanup()')
       // for healthcheck
       this._tags = metadata.vm.tags
+      this._hasTransferredData = true
+    }
+    if (transferList.length === 0) {
+      this._hasTransferredData = false
```

Comment:
I would write it like this 
```suggestion
      this._hasTransferredData = transferList.length > 0 
```
check if the parenthesis are ok, the suggestion lines seems  strange